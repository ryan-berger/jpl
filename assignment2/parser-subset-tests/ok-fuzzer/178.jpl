show 7
assert 3, "308"
write image 5 to ")4#"
write image 3 to "2{"
show I(C)
write image 349 to ""
read image "V" to H1c.
time read image "" to i1E.
read image "zY" to B
time assert Q0j.(59, 3.50), "wXc"
print "R"
print "mN"
return o(b())
show p(816, .11, M(2., Z(.91924344619403331376562), Q0g., .52160, W1U.(1, 5993.941791750166, g, S, 09, 1., p, N(99118855), Q6z.9l.3e.)), Q8k_0u.(.373, 3.8, 33.6, .03))
write image .018 to "+"
