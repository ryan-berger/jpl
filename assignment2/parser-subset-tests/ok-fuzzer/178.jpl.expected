(ShowCmd (IntExpr 7))
(StmtCmd (AssertStmt (IntExpr 3) "308"))
(WriteImageCmd (IntExpr 5) ")4#")
(WriteImageCmd (IntExpr 3) "2{")
(ShowCmd (CallExpr I (VarExpr C)))
(WriteImageCmd (IntExpr 349) "")
(ReadImageCmd "V" (VarArgument H1c.))
(TimeCmd (ReadImageCmd "" (VarArgument i1E.)))
(ReadImageCmd "zY" (VarArgument B))
(TimeCmd
 (StmtCmd (AssertStmt (CallExpr Q0j. (IntExpr 59) (FloatExpr 3)) "wXc")))
(PrintCmd "R")
(PrintCmd "mN")
(StmtCmd (ReturnStmt (CallExpr o (CallExpr b))))
(ShowCmd
 (CallExpr
  p
  (IntExpr 816)
  (FloatExpr 0)
  (CallExpr
   M
   (FloatExpr 2)
   (CallExpr Z (FloatExpr 0))
   (VarExpr Q0g.)
   (FloatExpr 0)
   (CallExpr
    W1U.
    (IntExpr 1)
    (FloatExpr 5993)
    (VarExpr g)
    (VarExpr S)
    (IntExpr 9)
    (FloatExpr 1)
    (VarExpr p)
    (CallExpr N (IntExpr 99118855))
    (VarExpr Q6z.9l.3e.)))
  (CallExpr Q8k_0u. (FloatExpr 0) (FloatExpr 3) (FloatExpr 33) (FloatExpr 0))))
(WriteImageCmd (FloatExpr 0) "+")
